platform/execution/src/WorkerService.js

```diff
@@ -66,7 +70,8 @@ define(
          */
```

Comment:
Being picky, but method now returns {Worker | SharedWorker}, so JSDoc probably needs updating. Might be worth mentioning as well when each is returned (ie. that a SharedWorker is specified using the 'shared' flag at declaration time)